easybuild/easyconfigs/p/PyBERT/PyBERT-1.7.9_2to3.patch

```diff
@@ -0,0 +1,415 @@
+# Python 2to3
```

Comment:
since `PyGIMLi` is also compatible with Python 2, there's probably no need for this?

it's hard to tell whether this broke anything, or whether this patch is complete...